src/shared/types/ErrorCode.ts

```diff
@@ -80,6 +80,9 @@ enum ErrorCode {
   // course.navMenuItem
   // > update
   NavItemNotFound = 'CAPI25',
-};
+
+  MigrationTimeout = 'CAPI26',
+  MigrationIssue = 'CAPI28',
```

Comment:
is there a reason we skip 27 here?

Reply:
or, not that you necessarily wrote all of these, a reason we go from 21 to 25 for NavItemNotFound

Reply:
They're just out of order it seems :) Looks good